### PR TITLE
WIP Adde provision to save entity for latitude and longitude

### PR DESCRIPTION
**Pull request by Edzelopez**

### CODE REVIEW

File: ao.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+ <?php
```
Comment (seamuslee001) [inline ao.php]: @Edzelopez looks like your editor issue at it again

File: ao.php [hunk 2]
```diff
@@ -232,6 +232,33 @@ function ao_civicrm_post( $op, $objectName, $objectId, &$objectRef ) {
     $objectRef->body_html = str_replace('[show_link]', 'https://www.autismontario.com/civicrm/mailing/view?reset=1&id=' . $objectId, $objectRef->body_html);
     $objectRef->save();
   }
+  if ($objectName == "Address" && $op == "create") {
+    // Get appropriate entity ID for address
+    // Can be either contact ID or event ID
+    $object = "Event"; // Can be Contact as well, depending on if the field is stored in the contact entity
+    $entityId = 1327; // Event ID or Contact ID.
+    $entityType = SupportedEntities::getEntityType($object);
```
Comment (seamuslee001) [inline ao.php]: Does this need to be name spaced as well?

**Review by seamuslee001 — APPROVED** [review]
Changes look correct based on the data stored in the table already

File: ao.php (hunk 1 above)
Comment (Edzelopez) [reply ao.php]: Yup, I figured what was wrong and corrected it.

**Pull request merged**